### PR TITLE
TransSensual own scraper and removed from RK

### PR DESCRIPTION
**Pull request by ZZenvo**
TransSensual didn't seem to work with the RealityKings scraper anymore.
So made it's own and deleted it from the RK scraper.

### CODE REVIEW

**Comment by ZZenvo** [comment]
Updated with your feedback.
Also decided to just add moviescraper part as well, but it is kinda shitty/pointless, because the movie section on their site don't make too much sense.

File: scrapers/TransSensual.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  # of some scenes they released with same theme? 
+  # So a movie has date as the earliest released scene, and other scenes have their own.
+  # Description is just name of the 'movie', so gonna go with whole 'desc & cat' as synopsis.
+  # So i'd say movies on TransSensual aren't 'proper'. But here goes:
+  movieScraper:
+    movie:
+      Name: //h2[@class='wxt7nk-4 fSsARZ']/text()
+#      Director: 
+#      Duration: 
+      Aliases:
```
Comment (bnkai) [inline scrapers/TransSensual.yml:49]: Instead of aliases it gets the performers. Is this intentional?

File: scrapers/TransSensual.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      Date:
+        selector: //span/div[contains(.,"Release Date")]/../text()
+        parseDate: January 2, 2006
+      Synopsis: 
+        selector: //div[@class='tjb798-2 flgKJM']//text()  # just to have something in synopsis
+        concat: " "
+      Studio:
+        Name: //meta[@name="dti.network"]/@content
+      FrontImage: //img[contains(@src, "cover")]/@src
+      BackImage: //img[contains(@src, "cover")]/@src
```
Comment (bnkai) [inline scrapers/TransSensual.yml:61]: Since there is no backimage shouldn't we just leave it out?

File: scrapers/TransSensual.yml (hunk 1 above)
Comment (ZZenvo) [reply scrapers/TransSensual.yml:49]: Yes, that was intentional.
Some movies got title-aliases?
Just liked having names in that field myself, but we can change it.
I guess with a movie section overhaul, some of these things will change anyways?

File: scrapers/TransSensual.yml (hunk 2 above)
Comment (ZZenvo) [reply scrapers/TransSensual.yml:61]: Yeah, forgot that movie-tab view only needed/shows frontimage, so we can leave it out.

**Comment by ZZenvo** [comment]
Seems like site has changed(classes names) because it is part of RK network.

**Comment by ZZenvo** [comment]
Updated with your feedback & retested with a few scenes and movies.
Scenescraper:
Supported Site|Scraper|Content
------------- | ------------- | -------------
transsensual.com/scene|TransSensual.yml|Trans

Moviescraper:
Supported Site|Scraper|Content
------------- | ------------- | -------------
transsensual.com/movie|TransSensual.yml|Trans

**Pull request merged**